core-bundle/src/Controller/BackendController.php

```diff
@@ -182,12 +181,7 @@ public function pickerAction(Request $request): RedirectResponse
         $extras = [];
 
         if ($request->query->has('extras')) {
-            if ($request->query instanceof InputBag) {
-                $extras = $request->query->all('extras');
-            } else { /** @phpstan-ignore-line */
-                // Backwards compatibility with symfony/http-foundation <5.0
-                $extras = $request->query->get('extras');
-            }
+            $extras = $request->query->all('extras');
 
             if (empty($extras) || !\is_array($extras)) {
```

Comment:
`InputBag#all()` always returns an array, so we can IMHO drop the `is_array` check.

Reply:
Removed in ad0ef7b3b5a99aea231805324931b5da23f59315.